c7n/resources/rds.py

```diff
@@ -290,17 +292,26 @@ def process(self, resources):
                                 f.exception()))
 
     def process_snapshot_retention(self, resource):
-        v = int(resource.get('BackupRetentionPeriod', 0))
-        if ((v == 0 or v < self.data['days']) and
+        current_retention = int(resource.get('BackupRetentionPeriod', 0))
+        current_copy_tags = resource['CopyTagsToSnapshot']
+        new_retention = self.data['days']
+        new_copy_tags = self.data.get('copy-tags', False)
```

Comment:
Just to keep backwards-compat - The default value when a DB Instance is created is **False**.
No problem with you changing the default to **True**.